embc-app/ClientApp/src/app/provincial-admin/components/admin-volunteer-maker/admin-volunteer-maker.component.ts

```diff
@@ -0,0 +1,355 @@
+import { Component, OnInit } from '@angular/core';
+import { ActivatedRoute, Router } from '@angular/router';
+import { FormBuilder, Validators, FormGroup } from '@angular/forms';
+import { combineLatest, Observable, of } from 'rxjs';
+import { switchMap } from 'rxjs/operators';
+import { AuthService } from 'src/app/core/services/auth.service';
+import { VolunteerService } from 'src/app/core/services/volunteer.service';
+import { OrganizationService } from 'src/app/core/services/organization.service';
+import { Volunteer, Organization, ListResult } from 'src/app/core/models';
+import { NotificationQueueService } from 'src/app/core/services/notification-queue.service';
+import { UniqueKeyService } from 'src/app/core/services/unique-key.service';
+import { invalidField } from 'src/app/shared/utils';
+
+@Component({
+  selector: 'app-admin-volunteer-maker',
+  templateUrl: './admin-volunteer-maker.component.html',
+  styleUrls: ['./admin-volunteer-maker.component.scss']
+})
+export class AdminVolunteerMakerComponent implements OnInit {
+
+
+  editing = true; // whether we are adding/editing or reviewing
+  submitting = false; // whether we are submitting data to BE
+  doSelectOrg: boolean = null; // whether we can select org (not set by default)
+  iAmLocalAuthority = false;
```

Comment:
Could you please delete this obsolete variable?
And also the code below.

Reply:
Doing it in another branch.